src/appleseed/renderer/modeling/camera/camera.h

```diff
@@ -171,26 +177,6 @@ class APPLESEED_DLLSYMBOL Camera
     float               m_inverse_cdf_close_point;
```

Comment:
I suggest to move inverse_cdf_close_point/inverse_cdf_open_point to impl as well, those two are internal vars used only for inverse sampling. Normalized times seem to be internal vars as well.

Reply:
Yes, makes sense.